lib/warder/cli/arguments.rb

```diff
@@ -44,6 +46,18 @@ def version(opts)
         end
       end
 
+      def quiet(opts)
+        opts.on('-q', '--quiet', 'Do not echo validators output.') do
+          @options['quiet'] = true
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.